src/main/java/net/mcreator/ui/workspace/resources/WorkspacePanelTextures.java

```diff
@@ -39,12 +39,12 @@
 
 import javax.swing.*;
 import java.awt.*;
-import java.awt.event.MouseAdapter;
-import java.awt.event.MouseEvent;
+import java.awt.event.*;
 import java.io.File;
 import java.util.List;
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Consumer;

```

Comment:
Don't think this import is needed

Reply:
I'm sorry, maybe I need to change my eclipse to Idea